src/utils/services/index.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+export async function getVideos(keyword) {
+  const searchBaseUrl = `https://www.googleapis.com/youtube/v3/search?maxResults=25&key=${process.env.REACT_APP_API_KEY}&part=snippet&q=${keyword}`;
+  console.log(searchBaseUrl);
```

Comment:
Are the console logs relevant for you in this file? If you don't need them anymore then consider deleting them.